### PR TITLE
google test: api: Migrate solver_black.

### PR DESCRIPTION
**Pull request by aniemetz**

### CODE REVIEW

File: test/unit/api/solver_black.cpp [hunk 1]
```diff
@@ -0,0 +1,2253 @@
...
+
+TEST_F(TestApiSolverBlack, getInfo)
+{
+  ASSERT_NO_THROW(d_solver.getInfo("name"));
+  ASSERT_THROW(d_solver.getInfo("asdf"), CVC4ApiException);
+}
+
+TEST_F(TestApiSolverBlack, getInterpolant)
+{
+  // TODO
```
Comment (mpreiner) [inline test/unit/api/solver_black.cpp:1250]: Who's TODO is this?

Comment (mpreiner) [reply test/unit/api/solver_black.cpp:1250]: Was introduced in https://github.com/CVC4/CVC4/pull/4638 . @yoni, @Ying1123

Comment (aniemetz) [reply test/unit/api/solver_black.cpp:1250]: I already pinged them offline, but doesn't hurt to ping them here, too :)

Comment (mpreiner) [reply test/unit/api/solver_black.cpp:1250]: Except that I pinged the wrong Yoni :)

**Review by mpreiner — APPROVED** [review]

File: test/unit/api/solver_black.cpp (hunk 1 above)
Comment (aniemetz) [reply test/unit/api/solver_black.cpp:1250]: I'll create an issue and will add the reference to the TODO.

Comment (aniemetz) [reply test/unit/api/solver_black.cpp]: #5593

**Pull request merged**